tutorials/physics/rigid_body.rst

```diff
@@ -0,0 +1,51 @@
+.. _doc_rigid_body
+
+Rigid Body
+=========
```

Comment:
To prevent build warnings, the heading delimiters must match the length of the section title.